src/spec/page-objects/billing-page.js

```diff
@@ -7,6 +7,7 @@ class BillingPage {
     this._ccIconAmex = $('.icon-amex');
     this._ccIconVisa = $('.icon-visa');
     this._ccIconMaster = $('.icon-master');
+    this._countrySelect =  element(by.id('country'));
```

Comment:
If `$('#` works fine, I prefer to keep that way, in order to have the same style (I think that, at the moment, we have more _css selector like_ ones than _by_ ones.

But, if it has any advantage, we can all switch to _by_ ones